Resources/Prototypes/Catalog/VendingMachines/Inventories/cart.yml

```diff
@@ -16,3 +16,6 @@
     EncryptionKeyScience: 2
     EncryptionKeySecurity: 1
     EncryptionKeyService: 3
+  contrabandInventory:
+    RubberStampGreytide: 1
```

Comment:
Meant to be a very rare maints item, and also weird that HoP would have it. Keep the pen, not the stamp.

Reply:
In my original design it was going to be a Syndicate Stamp, as sort of a trophy for being able to break into the HoP's office and hack their vending machine, but there was general protest in my discussions about this because those sell for 2TC.

I wanted there to be some sort of untraceable trophy in this machine for being able to pull it off, but perhaps it's too much.

Reply:
As seen with the Sec clothesvend and the funny clown mask, the people who are most likely to hack open a vending machine are the people who have direct access to it. 

Maybe an Ian and Nanotrasen balloon, since it's in HoP's office?

Reply:
I dig it, good ideas there.

Reply:
Swapped the Greytider Stamp for 2 NT Balloons. I really like this one for the general vibe of what the HoP is going for.